### ISSUE TITLE
Proposal: Add a Fixtures guide

### ISSUE THREAD

**Issue by chasenlehara** [canjs/canjs#5046]
__TLDR:__ add docs on what fixtures are, how they work, and how they can improve the development workflow.

## Motivation

Fixtures are a game-changing tech that can have a significant impact on development and testing. We make extensive use of them in our examples. We also sometimes get [asked if we have a guide on setting up fixtures](https://bitovi-community.slack.com/archives/CFC22NZ8A/p1558538698022300).

It’d be nice to have a fixtures guide that we can 1) point to as a best practice and 2) link to in our docs wherever we use fixtures.

## Solution

Let’s create a guide that:

- Discusses the value of fixtures for developing and testing
- Has examples of using fixtures
- Includes best practices for how to set up fixtures and get the most out of them

## Prior art

- [AJAX fixtures for JavaScript](https://www.bitovi.com/blog/ajax-fixtures-javascript)
- [CanJS v2 Data Models and Fixtures](https://v2.canjs.com/guides/DataModelsAndFixtures.html)
